aws/resource_aws_apigatewayv2_integration.go

```diff
@@ -102,10 +102,17 @@ func resourceAwsApiGatewayV2Integration() *schema.Resource {
 					"2.0",
 				}, false),
 			},
+			"request_parameters": {
+				Type:     schema.TypeMap,
+				Optional: true,
+				// Length between [1-512].
+				Elem: &schema.Schema{Type: schema.TypeString},
```

Comment:
can we add the validation rule here? or is there an issue with type map and validations?

Reply:
Once Plugin SDK v2 support is merged we can use https://github.com/hashicorp/terraform-plugin-sdk/pull/304.

Reply:
cool